cmd/fortifyExecuteScan.go

```diff
@@ -182,7 +200,7 @@ func runFortifyScan(config fortifyExecuteScanOptions, sys fortify.System, comman
 	influx.fortify_data.fields.projectName = fortifyProjectName
 	influx.fortify_data.fields.projectVersion = fortifyProjectVersion
 	influx.fortify_data.fields.violations = fmt.Sprintf("%v", numberOfViolations)
-	if numberOfViolations > 0 {
+	if numberOfViolations > 0 && config.ConsiderSuspicious {
```

Comment:
That's not the intention of `considerSuspicious`, sorry but I have a very strong opinion on this one.